iio-config.h.cmakein

```diff
@@ -17,9 +17,9 @@
 #cmakedefine01 WITH_NETWORK_EVENTFD
 #cmakedefine01 WITH_IIOD_USBD
 #cmakedefine01 WITH_LOCAL_CONFIG
+#cmakedefine01 WITH_AIO
```

Comment:
Just a comment which has noting to do with this... We should definitely thing about replacing aio for io_uring and liburing  for libiio2 :smile:

Reply:
Yes, I will definitely have a look at io_uring. I want to see if it can help us with zero-copy.

Reply:
It has both ring buffers shared between kernel and userspace so that's something already in the right way... Another neat thing is that we can potential reduce a lot the number of syscalls.

Reply:
But I assume the IIO subsystem needs to support it (does it?).

Reply:
Don't think so. At least for normal reads/writes on sysfs stuff. Now for ioctl (not sure if that stuff was merged; and yes, it seems io_uring is also allowing to re-implement ioctl), the last time I checked I think IIO would have to support it...

Reply:
https://lwn.net/Articles/844875/ - not idea the outcome of the RFC